ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -1,12 +1,82 @@
 from styx_msgs.msg import TrafficLight
+import tensorflow as tf
+import numpy as np
+import datetime
+
+from PIL import Image
+
 
 class TLClassifier(object):
+    """
+    This class uses the trained model from the TensorFlow Object Detection API.
+    https://github.com/tensorflow/models/tree/master/research/object_detection
+
+    For the Capstone Project, a Single Shot Detector with lightweight MobileNet
+    Backbone was trained on the Bosch Traffic Light Dataset, LISA Traffic Light Dataset and
+    Udacity Simulator and Site images.
+
+    The Inference Code was adapted from:
+    https://github.com/tensorflow/models/blob/master/research/object_detection/inference/detection_inference.py
+
+    """
     def __init__(self):
-        #TODO load classifier
-        pass
+        # Threshold for detections
+        self.detection_threshold = 0.5
+
+        # Path to frozen Graph
+        graph_path = 'src/tl_detector/light_classification/inference_graph/frozen_inference_graph.pb'
+
+        # Create the TensorFlow session in which the graph is loaded
+        self.session = tf.Session()
```

Comment:
Apparently you can add \_\_exit\_\_ to clean up the tensorflow session on exit. Not sure how necessary this is, and if it needs to be combined with any ROS lifecycle?